readme.md

```diff
@@ -303,6 +303,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [SSH](https://github.com/moul/awesome-ssh)
 - [FOSS for Developers](https://github.com/httpsGithubParty/FOSS-for-Dev)
 - [HyperTerm](https://github.com/bnb/awesome-hyperterm)
+- [PowerShell](https://github.com/janikvonrotz/awesome-powershell)
```

Comment:
Add a succinct description. See other items with description for how to format it.